tests/model_serving/model_server/authentication/conftest.py

```diff
@@ -127,14 +164,31 @@ def patched_remove_authentication_isvc(
         patches={
             http_s3_caikit_serverless_inference_service: {
                 "metadata": {
-                    "annotations": {"security.opendatahub.io/enable-auth": "false"},
+                    "annotations": {Annotations.KserveAuth.SECURITY: "false"},
```

Comment:
all places should be updated with the relevant constant

Reply:
ack